src/Context.php

```diff
@@ -142,28 +149,21 @@ public function setAttribute(string $key, $value): self
     }
 
     /**
+     * @param ExclusionStrategyInterface|ValueExclusionStrategyInterface $strategy
+     *
      * @return $this
      */
-    public function addExclusionStrategy(ExclusionStrategyInterface $strategy): self
+    public function addExclusionStrategy($strategy): self
```

Comment:
I changed the strong type hint for the argument. I annotated accordingly 

But the argument validation is performed in the `$this->exclusionStrategy->addStrategy()`